spec/index.bs

```diff
@@ -71,14 +77,37 @@ dictionary GPUExtent3DDict {
 typedef (sequence<u32> or GPUExtent3DDict) GPUExtent3D;
 </script>
 
-## Base Objects ## {#base-objects}
+## Internal Objects ## {#webgpu-internal-objects}
```

Comment:
Internal objects exist to be shared across "agents" (threads). IDL interfaces cannot, because they are JS objects.

Reply:
I wouldn’t call them “internal,” then.

Reply:
(sorry for the slow response.) Can you elaborate - why not? They are not exposed to the web platform - that's why they're called internal (in line with "internal slot"). Should a different name be used?

Reply:
> It seems confusing to me: it implies that some objects are not internal.

To clarify, the interface objects are not internal.

> Moreover, what is the value in saying that an interface exposes API to an internal object? I mean, how is it internal any more as we have a public interface to it...

I was imagining the tracked state as being internal to the internal object, so it would expose the API. However I realize now that doesn't make sense - the internal object should be like a "dumb struct" whose state tracking is visible to the GPU* object which exposes it.